requirements/dev.txt

```diff
@@ -1,4 +1,5 @@
 numpy
 aiohttp
-cffi==1.15.0
-psutil==5.9.4
\ No newline at end of file
+cffi==1.16.0
+psutil==5.9.4
+setuptools==70.2.0
```

Comment:
Can you also add small descr into CHANGES.txt and increment to minot version in setup.py: `version='1.11.7'` ?

Reply:
Sure! Added